tests/test_verilog_conversion.py

```diff
@@ -5,7 +5,7 @@
 from hdlConvertorAst.hdlAst import HdlModuleDec, HdlModuleDef, HdlDirection
 from hdlConvertorAst.language import Language
 
-from tests.hdl_parse_tc import TEST_DIR, HdlParseTC, parseFile
+from hdl_parse_tc import TEST_DIR, HdlParseTC, parseFile
```

Comment:
use `python3 -m tests...` instead of `python3 tests/...` in order to have imports working

Reply:
Also, are you able to rewrite this commit that this will not end up in history?
(e.g. `git reset --soft HEAD~1`, rm change, git add, commit, push -f)

Reply:
ok, I will rewrite this commit, but I don't know how to generate/write an exact correct reference file.

Reply:
I don't know how to modify this pull request yet, I just replace this by #126 instead.